package.json

```diff
@@ -23,10 +23,11 @@
     "test-single": "karma start --single-run --browsers PhantomJS"
   },
   "dependencies": {
+    "@frctl/fractal": "^1.0.11",
```

Comment:
This looks like an accidental inclusion. We don't need to include fractal in fec-style since it's its own repo.